spec/assessments/transitionWordsSpec.js

```diff
@@ -43,7 +47,8 @@ describe( "An assessment for transition word percentage", function(){
 			transitionWordSentences: 2 } ), i18n );
 
 		expect( assessment.getScore() ).toEqual( 9 );
-		expect( assessment.getText() ).toEqual ( "66.7% of the sentences contain a transition word or phrase, which is great." );
+		expect( assessment.getText() ).toEqual ( "66.7% of the sentences contain a <a href='https://yoa.st/transition-words' target='_blank'>transition word</a> " +
```

Comment:
For tests this is less important, but why not create a variable for the repeating text? This makes the code less complex and if the text changes you only have to change this at one place instead of 6+.